src/main/java/org/gridsuite/ds/server/utils/SetGroupType.java

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.ds.server.utils;
+
+/**
+ * @author Thang PHAM <quyet-thang.pham at rte-france.com>
+ */
+public enum SetGroupType {
+    FIXED,
+    PREFIX,
+    SUFFIX
+}
```

Comment:
I failed to understand what is the point of a SetGroupType. The enum looks more like a relative position than a type. I'd like you to explain it to me (and maybe add a comment or whatever).

Reply:
DONE in https://github.com/gridsuite/dynamic-simulation-server/pull/95/commits/3f2f55b576f7e0b3ade71c709a55bcbdd3572e9e